cgmes/cgmes-measurements/src/main/java/com/powsybl/cgmes/measurements/CgmesAnalogPostProcessor.java

```diff
@@ -0,0 +1,141 @@
+        if (bay != null) {
+            String voltageLevelId = bay.getId("VoltageLevel");
+            LOG.info("Power resource system {} of Analog {} is a Bay: Analog is attached to the associated voltage level {}",
+                    powerSystemResourceId, id, voltageLevelId);
+            VoltageLevel voltageLevel = network.getVoltageLevel(voltageLevelId);
+            if (voltageLevel == null) {
+                LOG.warn("Ignored {} {}: associated voltage level {} not found", measurementType, id, voltageLevelId);
+                return;
+            }
+            voltageLevel.setProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "Analog_" + measurementType, id);
+        } else {
+            LOG.warn("Ignored {} {}: attached power system resource {} not found", measurementType, id, powerSystemResourceId);
+        }
+    }
+
+    private static void createMeas(Identifiable<?> identifiable, String id, String terminalId, String measurementType, Map<String, String> typesMapping) {
+        if (identifiable instanceof Connectable) {
+            Connectable<?> c = (Connectable<?>) identifiable;
+            Measurements meas = c.getExtension(Measurements.class);
+            if (meas == null) {
+                c.newExtension(MeasurementsAdder.class).add();
+                meas = c.getExtension(Measurements.class);
+            }
+            Measurement.Type type = getType(measurementType, typesMapping);
+            Measurement.Side side = null;
+            MeasurementAdder adder = meas.newMeasurement()
+                    .setValid(false)
+                    .setId(id);
+            if (!(c instanceof Injection)) {
+                side = getSide(terminalId, c);
+            }
+            if (type != OTHER && side == null && !(c instanceof Injection)) {
+                adder.setType(OTHER);
+            } else {
+                adder.setType(type);
+            }
+            adder.setSide(side);
+            Measurement measurement = adder.add();
+            measurement.putProperty("cgmesType", measurementType);
+        } else {
+            identifiable.setProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "Analog_" + measurementType, id);
+        }
+        // TODO get value of measurement
+    }
+
+    private static Measurement.Type getType(String measurementType, Map<String, String> typesMapping) {
+        switch (measurementType) {
+            case "ThreePhaseActivePower":
+            case "ActivePower":
+                return ACTIVE_POWER;
+            case "Angle":
+                return ANGLE;
+            case "ApparentPower":
+                return APPARENT_POWER;
+            case "LineCurrent":
+            case "Current":
+                return CURRENT;
+            case "Frequency":
+                return FREQUENCY;
+            case "ThreePhaseReactivePower":
+            case "ReactivePower":
+                return REACTIVE_POWER;
+            case "LineToLineVoltage":
```

Comment:
Here I am not sure that we should support `LineToLineVoltage` type because for me it is a wrong mapping.